### PR TITLE
updated free iOS book

### PR DESCRIPTION
**Pull request by amannayak0007**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by metabiswadeep** [comment]
Write name in alphabetical order to avoid linter error.

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -961,6 +961,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [NSHipster](http://nshipster.com/#archive) (Resource)
 * [Start Developing iOS Apps (Swift)](https://developer.apple.com/library/prerelease/ios/referencelibrary/GettingStarted/DevelopiOSAppsSwift/index.html) (HTML)
 * [Start Developing iOS Apps Today (Objective-C) - Last updated 22.10.2013](http://everythingcomputerscience.com/books/RoadMapiOS.pdf) (PDF)
+* [Develop in Swift Explorations](https://books.apple.com/in/book/develop-in-swift-explorations/id1581182728) - (iBook)
```
Comment (metabiswadeep) [inline books/free-programming-books-langs.md:964]: Move line 964 to line 958.

**Review by metabiswadeep — CHANGES_REQUESTED** [review]
Move line 964 to line 958.

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-langs.md:964]: complete with creators

```suggestion
* [Develop in Swift Explorations](https://books.apple.com/in/book/develop-in-swift-explorations/id1581182728) - Apple Education (iBook)
```

**Review by davorpa — APPROVED** [review]

**Review by metabiswadeep — APPROVED** [review]
Fine from my side.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**